app/helpers/assessment_detail_helper.rb

```diff
@@ -20,4 +20,17 @@ def updated_neighbour_responses_summary_text(responses, detail)
 
     t(:neighbour_responses_by_summary_tag, tag: "neighbour response", count:)
   end
+
+  def summary_advice_content(summary_tag)
+    case summary_tag.to_sym
+    when :complies
+      {color: "green", subheading: "Likely to be supported", message: "Your proposal is likely to be supported based on the information you have provided."}
+    when :needs_changes
+      {color: "orange", subheading: "Likely to be supported with changes", message: "Your proposal is generally acceptable, provided you address the recommended changes in this report."}
+    when :does_not_comply
+      {color: "red", subheading: "Unlikely to be supported", message: "Your proposal includes elements that do not comply with current planning policy."}
+    else
+      {color: "grey", subheading: "Notice", message: "No further information provided."}
+    end
+  end
```

Comment:
Minor, but should these go in the translations file?

Reply:
```suggestion
  def summary_advice_content(summary_tag)
    case summary_tag.to_sym
    when :complies
      {colour: "green", subheading: "Likely to be supported", message: "Your proposal is likely to be supported based on the information you have provided."}
    when :needs_changes
      {colour: "orange", subheading: "Likely to be supported with changes", message: "Your proposal is generally acceptable, provided you address the recommended changes in this report."}
    when :does_not_comply
      {colour: "red", subheading: "Unlikely to be supported", message: "Your proposal includes elements that do not comply with current planning policy."}
    else
      {colour: "grey", subheading: "Notice", message: "No further information provided."}
    end
  end
```

If these match the parameters needed by the component I think you can just use `**content` in app/views/planning_applications/assessment/assessment_details/summary_of_advice/show.html.erb